### PR TITLE
Adding new Ansible checks

### PR DESCRIPTION
**Pull request by dokuhebi**
After years of observation, I think I have enough knowledge of Ansible to help out writing those checks.  I added a new check for the sshd ClientAliveIntervalMax, but want to make sure this is what is needed before I start plugging away at more.

### CODE REVIEW

**Comment by mpreisler** [comment]
@openscap-jenkins add to whitelist

File: shared/templates/static/ansible/sshd_set_keepalive.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: "Set SSH Maximum Idle Keepalive Count"
+  lineinfile:
+    create=yes
+    dest="/etc/ssh/sshd_config"
+    regexp="^ClientAliveIntervalMax\s"
+    line="ClientAliveIntervalMax (ansible-populate sshd_set_keepalive)"
```
Comment (mpreisler) [inline shared/templates/static/ansible/sshd_set_keepalive.yml]: Shouldn't this use ClientAliveCountMax?

Comment (mpreisler) [inline shared/templates/static/ansible/sshd_set_keepalive.yml]: Is the XCCDF Value for `(ansible-populate sshd_set_keepalive)` declared anywhere? I can't find it in `shared/xccdf`.

**Comment by mpreisler** [comment]
Jenkins error:
```
File '/home/jenkins/workspace/scap-security-guide-pull-requests/label/node-el7/build/ssg-fedora-ds.xml' line 26185: Element '{http://checklists.nist.gov/xccdf/1.2}sub': No match found for key-sequence ['xccdf_org.ssgproject.content_rule_sshd_set_keepalive'] of keyref '{http://checklists.nist.gov/xccdf/1.2}subValueKeyRef'.
OpenSCAP Error: Invalid SCAP Source Datastream (1.2) content in /home/jenkins/workspace/scap-security-guide-pull-requests/label/node-el7/build/ssg-fedora-ds.xml. [oscap_source.c:319]
```

**Comment by shawndwells** [comment]
Hey Tom! Welcome, and thanks for jumping (back?) in :)

Since you mentioned wanting to make sure your approach is correct, there's a lot of things right:

- Naming the Ansible .yml file the same as the XCCDF rule will have the build system place it within the ``<fix>`` tag;
- Overall format of the ansible task is good

A few tweaks:

- The Ansible ``- name`` should match the XCCDF ``<title>``. For ``sshd_set_keepalive`` that would be ``Set SSH Client Alive Count``. Ref: [https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/xccdf/services/ssh.xml#L470](https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/xccdf/services/ssh.xml#L470) 
- Simple typo ``ClientAliveIntervalMax`` should be ``ClientAliveCountMax``
- There is no variable for this check, should be hard set to ``0``. The <description> tag _should_ generally indicate if a variable is used. What will always indicate this would be the ``<oval...>`` tag. 

Compare [sshd_set_keepalive](https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/xccdf/services/ssh.xml#L486):

``<oval id="sshd_set_keepalive" />``

vs [sshd_set_max_auth_tries](https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/xccdf/services/ssh.xml#L508):

``<oval id="sshd_set_max_auth_tries" value="sshd_max_auth_tries_value"/>``

Once those are cleaned up, this should be good to go.

**Comment by shawndwells** [comment]
@dokuhebi - Also, noticed you're working off your master branch. You may find branching easier, especially if you begin to work on multiple things at once. The "SSG on GitHub Primer" may be helpful: https://github.com/OpenSCAP/scap-security-guide/wiki/SSG-on-GitHub-Primer

**Comment by mpreisler** [comment]
This seems to be inactive / abandoned.

**Pull request closed without merging**